### ISSUE TITLE
Displaying Tweets

### ISSUE THREAD

**Issue by OriAboodi** [NWChen/banshee#15]
Hey @pr 
So you should be getting the tweets being returned in the format we discussed before. 
Can you display them in a nice way (any way that is not raw and illegible)?
Thanks!

**Comment by pr** [comment]
@Ori I'll give it my best shot right now. Currently, are the results displaying in a messy form or not at all?

**Comment by OriAboodi** [comment]
@pr unfortunately not at all, since Neil is still working on connecting to the new interface. Any chance you can get the framework up and running for this and hopefully we'll have enough time to test it tomorrow before class? It shouldn't be too hard, pretty self explanatory with the dictionary/json format being returned

**Comment by pr** [comment]
@ori understood. Can I still see if my part works without Neil's part being done? Not quite sure how to test, I've not done anything like this before. I think it's going to be awesome when it works.

**Comment by OriAboodi** [comment]
@pr definitely! We're all figuring this out as we go, I'm petty excited about the back-end stuff as well. Year you can test your part by passing into it some data with the same format I think. Great job trying out these new things.

**Comment by pr** [comment]
You too Ori, we got this. I'll update you tomorrow.

**Comment by pr** [comment]
@OriAboodi @NWChen 

Ok so I think I've gone about as far as I can go without Neil's part. 

I think we're going to end up using [Mustache](http://mustache.github.io/#demo
), or something like it, to turn the JSON into html. I updated the results page with my best guess of what that is going to look like, but I can't really test it for now. It's not correct, but I think it's close. 

Here what the mockup looks like, this is what I was shooting for. [peter.net/mockup ](https://peter.net/mockup)Do you want different information displayed about the tweets? Or in a different format? Let me know!

**Comment by NWChen** [comment]
Mockup looks excellent to me @pr .

**Comment by pr** [comment]
@NWChen Thanks! Do you want to display stats? The user can't currently query with them, so I'm not sure it's super necessary. 

Also if we can't tie it all together by class, we could use the mockup to give people a sense of where we're going.

**Comment by OriAboodi** [comment]
@pr It would be nice to display stats I think, it gives the tweet an extra layer. But if you can't pull it together by class, that's alright, as long as the rest of the tweet is displayed correctly.

The Mockup looks great btw

**Comment by pr** [comment]
Oh ok I'll add that, no problem. I don't think my part and Neil's part are integrated yet though.

**Comment by NWChen** [comment]
@pr I'm working on rendering highlighted keywords on the clientside, so will be making my changes exclusively in `background-scripts`. I think you've prepared everything needed for me to render these contexts, so I'll let the team know once I'm finished + will push my version up to Heroku upon completion.

**Comment by pr** [comment]
@NWChen Awesome, great work! If you need me to do anything else besides the retweets and likes let me know.
